serialization.rst

```diff
@@ -333,13 +333,16 @@ iterate over the array and set the properties without modifications. It
       -> [ "foo" => "yes", "__pclass" => Binary(0x80, "MyClass") ]
 
     { "foo": "yes", "__pclass" : { "$type": "80", "$binary": "OurClass" } }
-      -> [ "foo" => "yes", "__pclass" => Binary(0x80, "OurClass") ]
+      -> OurClass { $foo => "yes", $__pclass => Binary(0x80, "OurClass"), $unserialized => true }
 
```

Comment:
I don't agree that a __pclass should override an "array" typemap. The user very specifically asked for an array here, so that's what we should return.